src/translation-helpers.ts

```diff
@@ -0,0 +1,58 @@
+export const ErrorCode = {
```

Comment:
This could be an enum, with PascalCase names

Reply:
I think it is a controversial issue. I am trying to avoid using enums according [official documentation](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums)